csharp-basics/exercises/Scooters/Scooters/RentalCompany.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Scooters
+{
+    public class RentalCompany : IRentalCompany
+    {
+        private string _name;
+        private ScooterService _scooterService = new ScooterService();
+
+        public RentalCompany(string CompanyName)
+        {
+            _name = CompanyName;
+        }
+
+        public ScooterService GetScooterService()
+        {
+            return _scooterService;
+        }
+
+        public string Name
+        {
+            get => _name;
+            set => _name = value;
+        }
+        public void StartRent(string id)
+        {
+            _scooterService.GetScooterById(id).StartRent();
+        }
+
+        public decimal EndRent(string id)
+        {
+            _scooterService.GetScooterById(id).StopRent();
+            return _scooterService.GetScooterById(id).GetLastRentalEntryPrice();
```

Comment:
šī nav scooter atbildība. 
2.. tu jau rindu augstāk dabūji scooter pēc ID.. kapēc to darīt 2x?